app/viewmodels/checkAnswers/CommodityCodeSummary.scala

```diff
@@ -45,7 +45,7 @@ object CommodityCodeSummary {
       key = "commodityCode.checkYourAnswersLabel",
       value = ValueViewModel(HtmlFormat.escape(value).toString),
       actions = Seq(
-        ActionItemViewModel("site.change", routes.CommodityCodeController.onPageLoadUpdate(mode, recordId).url)
+        ActionItemViewModel("site.change", routes.HasCommodityCodeChangeController.onPageLoad(mode, recordId).url)
```

Comment:
this produces strange behaviour when the user clicks the change link at the end of this journey before they save their new answer - it goes back to the warning page - i think it should go back to the commodity code page - but double check with amaka

Reply:
The behaviour should be alright now